test/parallel/proofOfStake.3rdPartyRewards.activation.check.test.ts

```diff
@@ -0,0 +1,378 @@
+/*
+ *
+ * @group 3rdPartyRewards
+ */
+import { getApi, initApi } from "../../utils/api";
+import { AssetWallet, User } from "../../utils/User";
+import { Keyring } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import { Assets } from "../../utils/Assets";
+import { Sudo } from "../../utils/sudo";
+import { Xyk } from "../../utils/xyk";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { ProofOfStake } from "../../utils/ProofOfStake";
+import "jest-extended";
+import { burnLiquidity, getLiquidityAssetId } from "../../utils/tx";
+import { BN_ZERO, signTx } from "@mangata-finance/sdk";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+import { ExtrinsicResult, waitForRewards } from "../../utils/eventListeners";
+
+let testUser: User;
+let sudo: User;
+
+let keyring: Keyring;
+let newToken: BN;
+let newToken2: BN;
+let newToken3: BN;
+let liqId: BN;
+let liqId2: BN;
+
+describe("Proof of stake tests", () => {
+  beforeAll(async () => {
+    try {
+      getApi();
+    } catch (e) {
+      await initApi();
+    }
+
+    keyring = new Keyring({ type: "sr25519" });
+    sudo = new User(keyring, getEnvironmentRequiredVars().sudo);
+  });
+
+  beforeEach(async () => {
+    [testUser] = setupUsers();
+    [newToken, newToken2, newToken3] = await Assets.setupUserWithCurrencies(
+      sudo,
+      [Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT, Assets.DEFAULT_AMOUNT],
+      sudo,
+      true,
+    );
+
+    await setupApi();
+    await Sudo.batchAsSudoFinalized(
+      Assets.FinalizeTge(),
+      Assets.initIssuance(),
+      Assets.mintToken(newToken2, sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(testUser, Assets.DEFAULT_AMOUNT.muln(2)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6).muln(2)),
+      Sudo.sudoAs(
+        sudo,
+        Xyk.createPool(
+          MGA_ASSET_ID,
+          Assets.DEFAULT_AMOUNT.muln(20e6),
+          newToken2,
+          Assets.DEFAULT_AMOUNT.muln(20e6),
+        ),
+      ),
+    );
+    liqId = await getLiquidityAssetId(MGA_ASSET_ID, newToken2);
```

Comment:
can you please rename it by: 
LiqIdMgaToken2? 
ITs quite difficult to understand the test when there are more than one liq token :)

Reply:
Renamed all liqId tokens